web_widget_image_download/README.rst

```diff
@@ -0,0 +1,59 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
```

Comment:
If you ever want to create an LGPL website addon (such as a theme) and you want to depend on this module, you'd be violating the license.